src/components/ADempiere/PanelDefinition/index.vue

```diff
@@ -47,37 +56,45 @@ export default defineComponent({
     })
 
     const componentRender = computed(() => {
-      if (['browser', 'process'].includes(props.panelType)) {
+      if (['browser', 'process', 'table', 'window'].includes(props.panelType)) {
```

Comment:
Please remove this, the panelType is a dynamic way for handle display and is not limited to browser, process or table

Reply:
How do you suggest to handle it?
I thought of it that way because with new functionalities concerning redefinition such as Sequence, Tree, Accounting... conditions will be added to load the respective panel.